.github/workflows/node.js.yml

```diff
@@ -166,117 +166,7 @@ jobs:
             username: ${{ secrets.HISTAC_USERNAME }}
             privateKey: ${{ secrets.HISTAC_KEY}}
 
-            port: ${{ secrets.HISTAC_PORT }}
-
-  dev_namibia:
-    runs-on: ubuntu-latest
-
-    needs: pre_release
-
-    if: ${{ github.ref == 'refs/heads/dev' }}
-
-    steps:
-      - uses: garygrossgarten/github-action-ssh@release
-        name: Run the Update MicroFronEnd Script for Namibia
-        with:
-            command: who && which node && cd /opt/microfrontends && /bin/bash update_microfrontends.sh
-            host: ${{ secrets.HISTAC_HOST }}
-            username: ${{ secrets.HISTAC_USERNAME }}
-            privateKey: ${{ secrets.HISTAC_KEY}}
-
-            port: ${{ secrets.HISTAC_NAMIBIA_PORT }}
-
-  ohri_dev_namibia:
-    runs-on: ubuntu-latest
-
-    needs: pre_release
-
-    if: ${{ github.ref == 'refs/heads/dev' }}
-
-    steps:
-      - uses: garygrossgarten/github-action-ssh@release
-        name: Run the Update MicroFronEnd Script for Namibia
-        with:
-            command: who && which node && cd /opt/frontend_modules && /bin/bash update_microfrontends.sh
-            host: ${{ secrets.HISTAC_DEV_NAMIBIA_HOST }}
-            username: ${{ secrets.HISTAC_USERNAME }}
-            privateKey: ${{ secrets.HISTAC_KEY}}
-
-            port: ${{ secrets.HISTAC_DEV_NAMIBIA_PORT }}
-
-  pre_demo:
-    runs-on: ubuntu-latest
-
-    needs: build
-
-    if: startsWith(github.ref, 'refs/tags/v')
-
-    steps:
-      - uses: actions/checkout@v3
-      - name: Download Artifacts
-        uses: actions/download-artifact@v3
-      - name: Use Node.js
-        uses: actions/setup-node@v3
-        with:
-          node-version: "16"
-          registry-url: "https://registry.npmjs.org"
-      - run: yarn install --immutable
-      - run: yarn run ci:publish
-        env:
-          NODE_AUTH_TOKEN: ${{ secrets.NODE_AUTH_TOKEN }}
-
-  pre_demo_export_concepts:
```

Comment:
It seems `sandbox_demo` depends on this job. @larslemos please make a follow up PR for it as the CI isn't running